app/move/controllers/confirmation.test.js

```diff
@@ -0,0 +1,74 @@
+const controller = require('./confirmation')
+
+const {
+  data: mockMove,
+} = require('../../../test/fixtures/api-client/move.get.deserialized.json')
```

Comment:
This has been deprecated and refactor in #221. 

I think it's better to just mock the mock object with the properties that are required by this file rather than pull in the whole fixture. Have a look at the commits in #221 for what I've done elsewhere.